src/App.tsx

```diff
@@ -1,4 +1,12 @@
-function App() {
-  return <div></div>;
+import AppContent from "./components/appcontent";
+
+function App ({ appContent }: { appContent: any }) {
+  return <div>
+    <AppContent
+      showSidePane
+      showNavbar
```

Comment:
is Showsidepane and shownavbar always true ?

Reply:
No. It depends on whether to show it or not. We can set to false to hide navbar or sidepane

Reply:
`We can set to false`.
In this exisiting code, it is always set to true.
Right ?
in the case, why are we passing it like this ?

Reply:
We can pass the page inside AppContent component's appContent prop.. And the showSidePane and showNavbar can be set as per the requirement